src/js/actions/MesosActions.js

```diff
@@ -0,0 +1,78 @@
+var semver = require("semver");
+
+var AppDispatcher = require("../AppDispatcher");
+var JSONPUtil = require("../helpers/JSONPUtil");
+var MesosEvents = require("../events/MesosEvents");
+
+var MesosActions = {
+  requestVersionInformation: function (host) {
+    JSONPUtil.request(`${host}/version`).then(
+      function (data) {
+        AppDispatcher.dispatch({
+          actionType: MesosEvents.REQUEST_VERSION_INFORMATION_COMPLETE,
+          data: data
+        });
+      },
+      function (error) {
```

Comment:
I believe the error function is only ever called for network failures, but not for non-2xx responses. Maybe we should cover those too?